rgw/v2/tests/s3_swift/reusables/bucket_policy_ops.py

```diff
@@ -155,8 +157,26 @@ def DeleteObject(**kw):
     log.info(f"objects_delete_response: {objects_delete_response}")
     objects_delete_status = classify_response(objects_delete_response)
     if effect == "Allow":
+        if objects_delete_response:
+            objects_deleted_count = len(objects_delete_response["Deleted"])
+            log.error(f"expected number of objects deleted: {objects_count}")
+            log.error(f"actual number of objects deleted: {objects_deleted_count}")
+            if objects_deleted_count != objects_count:
```

Comment:
can we use ternary operator change code as bellow :
objects_delete_status = False if objects_deleted_count != objects_count else True